src/pages/LegacyIntegrationNotification.vue

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+import Vue from 'vue';
+import { ipcRenderer } from 'electron';
+import paths from '@/utils/paths';
+
+export default Vue.extend({
+  layout:     'dialog',
+  computed: {
+    oldIntegrationPath() {
+      return paths.oldIntegration;
+    },
+  },
+  mounted() {
+    ipcRenderer.send('dialog/ready');
+  },
+  methods: {
+    close() {
+      window.close();
+    }
+  },
+});
+</script>
+
+<template>
+  <div>
+    <h3>{{ t('legacyIntegrations.title') }}</h3>
+    <p>
+      {{ t('legacyIntegrations.messageFirstPart') }}
+      <code>{{ oldIntegrationPath }}</code>
+      {{ t('legacyIntegrations.messageSecondPart') }}
```

Comment:
A line break to separate the first and second part of the message might help to better communicate that an action is required on the user's part.